providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/resolvers/process/custom_ops.py

```diff
@@ -10,6 +10,12 @@
 logger = logging.getLogger("openfeature.contrib")
 
 
+class Fraction:
+    weight: int = 1
+    variant: str
+    pass
```

Comment:
nit - is `pass` needed here ?

Reply:
i dont thing so, i will adapt the code also based on @gruebel 's suggestion using dataclass